content/influxdb/cloud/tools/grafana.md

```diff
@@ -70,35 +70,74 @@ configure your InfluxDB connection:
 {{% tab-content %}}
 
 ## Configure Grafana to use InfluxQL
+To query InfluxDB Cloud from Grafana using InfluxQL:
 
-With **InfluxQL** selected as the query language in your InfluxDB data source,
-configure your InfluxDB connection:
+1. [Create an InfluxDB DBRP mapping](#create-an-influxdb-dbrp-mapping)
+2. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
+
+### Create an InfluxDB DBRP mapping
+When using InfluxQL to query InfluxDB Cloud, the query must specify a database and a retention policy.
+Use the [`influx v1 dbrp create` command](/influxdb/cloud/reference/cli/influx/v1/dbrp/create/)
```

Comment:
So this requires that the user have InfluxDB CLI downloaded and installed.  Is this requirement called out somewhere else (ideally a place every user would go by)?  I don't think I've seen this explicitly written in the docs (it may exist, just haven't seen it) and I think it would be useful for users of this doc itself.

Reply:
like...you will need the InfluxCLI to complete these steps.  Here is where you can find it -- and point at the CLI-only download. (as opposed to the bundled OSS/CLI combo pack!)

Reply:
I'll add a section about downloading the `influx` CLI.